MAUI/DataGrid/Column-types.md

```diff
@@ -894,6 +894,39 @@ dataGrid.Columns.Add(numericColumn);
 {% endhighlight %}
 {% endtabs %}
 
+## Allow Null
+
+This AllowNull property enhances the flexibility and usability of the application system by enabling users to input null values into numeric columns during the end edit phase. 
```

Comment:
The `DataGridNumericColumn.AllowNull` property allows you to commit a null value to the respective cell during the end edit.